src/server.rs

```diff
@@ -36,12 +43,13 @@ impl Server {
     pub fn random_port<A: AsRef<str>, P: AsRef<Path>>(
         ip_addr: A,
         serve_from: P,
+        retransmission_config: Option<RetransmissionConfig>,
```

Comment:
```suggestion
    pub fn random_port<A: AsRef<str>, P: AsRef<Path>>(
        ip_addr: A,
        serve_from: P,
```

Same feedback as above